src/components/router/components/Routes.js

```diff
@@ -18,14 +17,8 @@ const Routes = () => (
             <Switch>
                 <Route path={HOME} exact strict component={home.components.Home} />
                 <Route path={MARKET} exact strict component={home.components.Home} />
-                <Route path={PROFILE} exact strict component={userProfile.components.UserProfile} />
-                <Route
-                    path={ADD_ITEM}
-                    exact
-                    strict
-                    component={addItem.components.AddItem}
-                />
-                <Route path={WHO_WE_ARE} exact strict component={aboutUs.components.AboutUs} />
+                <Route path={PROFILE} component={userProfile.components.UserProfile} />
```

Comment:
Why did you remove `exact` and `strict`

Reply:
<img width="489" alt="screen shot 2019-02-03 at 9 54 20 am" src="https://user-images.githubusercontent.com/29875125/52180325-c3cd4500-2799-11e9-8978-a9e806ba4886.png">
Per the documentation, if i want to have other paths that match `/profile` and display within the `UserProfile` component (ex: `/profile/items`) i have to remove exact. Given how our routing is set up, all requests to anything starting with `/profile` will only match to `Userprofile` and nothing else. Secondly, `strict` isn't really needed here, it has no affect because we have additional url segments in the pathname. I actually think we can get rid of it for most of the routes, but not exactly sure.